weaviate/collection/classes/config.py

```diff
@@ -353,13 +360,13 @@ class Multi2VecClipConfig(VectorizerConfig):
 
 
 class Multi2VecBindConfigWeights(ConfigCreateModel):
-    audioFields: Optional[List[confloat(ge=0, le=1)]] = Field(None, alias="audio_fields")
-    depthFields: Optional[List[confloat(ge=0, le=1)]] = Field(None, alias="depth_fields")
-    imageFields: Optional[List[confloat(ge=0, le=1)]] = Field(None, alias="image_fields")
-    IMUFields: Optional[List[confloat(ge=0, le=1)]] = Field(None, alias="imu_fields")
-    textFields: Optional[List[confloat(ge=0, le=1)]] = Field(None, alias="text_fields")
-    thermalFields: Optional[List[confloat(ge=0, le=1)]] = Field(None, alias="thermal_fields")
-    videoFields: Optional[List[confloat(ge=0, le=1)]] = Field(None, alias="video_fields")
+    audioFields: Optional[List[float]] = Field(None, alias="audio_fields")
+    depthFields: Optional[List[float]] = Field(None, alias="depth_fields")
+    imageFields: Optional[List[float]] = Field(None, alias="image_fields")
+    IMUFields: Optional[List[float]] = Field(None, alias="imu_fields")
+    textFields: Optional[List[float]] = Field(None, alias="text_fields")
+    thermalFields: Optional[List[float]] = Field(None, alias="thermal_fields")
+    videoFields: Optional[List[float]] = Field(None, alias="video_fields")
```

Comment:
you can put these constraints into Fields: https://docs.pydantic.dev/latest/usage/fields/#numeric-constraints

Reply:
Yeh, I'd hoped so too but I couldn't think of a way to do this with `List[float]`. Using `f: float = Field(ge=0, lt=1)` makes sense but `lf: List[float] = Field(ge=0, lt=1)` sadly doesn't (Pydantic throws compile errors on it!)

Reply:
bummer